addons/binding/org.openhab.binding.avmfritz/ESH-INF/thing/thing-types.xml

```diff
@@ -176,4 +228,77 @@
 		<state readOnly="false"/>
     </channel-type>
 
+    <channel-type id="actual_temp" advanced="true">
+        <item-type>Number</item-type>
+        <label>Actual Temperature of heating thermostat</label>
+        <description>Actual measured temperature</description>
+        <category>Temperature</category>
+        <tags>
+            <tag>heating</tag>
```

Comment:
We haven't yet defined a tag library and thus "heating" is unknown - I would suggest to remove the tags from all of these channels for now.